src/System.Security.Cryptography.Primitives/tests/System.Security.Cryptography.Primitives.Tests.builds

```diff
@@ -4,8 +4,8 @@
   <ItemGroup>
     <Project Include="System.Security.Cryptography.Primitives.Tests.csproj"/>
     <Project Include="System.Security.Cryptography.Primitives.Tests.csproj">
-      <TargetGroup>netstandard1.7</TargetGroup>
-      <TestTFMs>netcoreapp1.1</TestTFMs>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
```

Comment:
Nit: we are trying to have the order of the metadata consistent across all of our test.builds files, so can you please sort the metdata in this order: TargetGroup, OSGroup, TestTFMs